source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -5,21 +5,245 @@ Enabling Translations on a New Repo
+workflow:
+
+#. If you are creating a new repository, use the `edx-cookiecutters`_ templates such as ``django-app`` or ``xblock``.
+
+#. For existing repos which don't have the ``make extract_translations`` command, it can be copied from the
+   `edx-cookiecutters`_ Makefile in the corresponding template e.g. ``cookiecutter-xblock`` Makefile for XBlocks.
+
+   **Note:** Some repositories use ``django-manage makemessages``. The recommendation is to use ``i18n_tool extract``
+   because it provides more options and cleans ``.po`` files.
+
+#. Run ``make extract_translations``. Verify it extracts files to the ``my_xblock_module/conf/locale``
+   directory. The ``.po`` filenames will vary depending on the use case:
+
+   - XBlocks: ``text.po``
+   - Django plugins: ``django.po``
+   - If the repo uses ``gettext`` and has a ``static`` directory with JavaScript, it may include ``djangojs.po``
+
+#. Add the repository to `extract-translation-source-files.yml`_ in the `openedx-translations repo`_.
+
+   Add a new entry under the ``python-translations`` section. For example for the XBlock located in
+   ``https://github.com/openedx/xblock-drag-and-drop-v2`` and has the XBlock Python 
+   module name ``drag_and_drop_v2`` it should have the following entry::
+
+    python-translations:
+      strategy:
+        matrix:
+          repo:
+            ...
+            - repo_name: xblock-drag-and-drop-v2
+              python_module_name: drag_and_drop_v2
+            ...
+
+   When adding a theme or any other module that's not an Open edX plugin or an XBlock the entry should
+   added without ``python_module_name`` as demonstrated below::
+
+    python-translations:
+      strategy:
+        matrix:
+          repo:
+            ...
+            - repo_name: credentials-themes
+            ...
+
+#. Create a draft pull request in the `openedx-translations repo`_
+
+#. Follow the instructions in the :ref:`testing-in-your-fork` section to verify new repository configuration.
+
+#. Mark your pull request as ready for review and wait for it to be merged.
+
+#. Verify the workflow is syncing the translations properly as described in the :ref:`debugging-translations` section.
+
+#. Install the XBlock or plugin in your local `Tutor`_ or `devstack`_ environment. Run
+   ``OPENEDX_ATLAS_PULL=true make pull_translations`` in the edx-platform repo to preview the translations.
+
+================================
+Django Microservice Repos (IDAs)
+================================
+
+The terms "Microservice" and "Independently Deployable Application (IDA)" are used interchangeably throughout the Open
+edX project. There are many Django microservices in the Open edX ecosystem, such as
+- `edx-platform`_ 
+- `credentials`_
+- `ecommerce`_
+- `course-discovery`_
+
+
+#. If you are creating a new repository, use the `cookiecutter-django-ida`_
+   template.
+
+**TODO:** How to enable translations infrastructure on a Django microservice
```

Comment:
Is this todo still relevant? Seems like you answer it below.

Reply:
Yes, it needs expansion and I'll follow up in other PRs.